service-config.json

```diff
@@ -0,0 +1,4 @@
+{ 
+  "Campaign": "http://localhost:7777/app",
+  "Community": "http://localhost:3006/app"
+}
```

Comment:
I can't comment on the templates/services/Campaign-server.js file, but you should .gitignore that.